.github/workflows/lint-pr.yml

```diff
@@ -166,7 +175,8 @@ jobs:
         with:
           files: |
             output1=info.txt
-            output2=history.txt
+            output2=history2.txt
```

Comment:
maybe a better file name to distinguish the git history that contains breaking changes history and features?

Reply:
thanks, updated